### PR TITLE
upload wheel to pypi

### PR DESCRIPTION
**Pull request by caishanli**
use ncnn's version string as pyncnn's version
test release to pypi

### CODE REVIEW

File: python/setup.py.i [hunk 1]
```diff
@@ -2,22 +2,43 @@ from setuptools import setup, find_packages
...
+  version           = "${PACKAGE_VERSION}",
+  author            = "nihui",
+  author_email      = "nihuini@tencent.com",
+  maintainer        = "caishanli",
+  maintainer_email  = "caishanli25@gmail.com",
+  description       = "ncnn is a high-performance neural network inference framework optimized for the mobile platform",
+  url               = "https://github.com/Tencent/ncnn",
+  classifiers       = [
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
```
Comment (nihui) [inline python/setup.py.i:35]: bsd 3-clause ?

**Review by nihui — CHANGES_REQUESTED** [review]

File: python/setup.py.i (hunk 1 above)
Comment (caishanli) [reply python/setup.py.i:35]: this is copied from xiaohanxiao's kears2ncnn......
```License :: OSI Approved :: BSD License```
is this correct?

Comment (caishanli) [reply python/setup.py.i:35]: all license is in [classifiers](https://pypi.org/classifiers/)

Comment (nihui) [reply python/setup.py.i:35]: BSD License is fine

**Pull request merged**